vispy/testing/_testing.py

```diff
@@ -141,7 +141,7 @@ def __exit__(self, exc_typ, exc, tb):
                                  self.exc.__name__)
         else:
             raise AssertionError("Expected %s, got %s instead (%s)" %
-                                 (self.exc.__name__, type(exc).__name__), exc)
+                                 (self.exc.__name__, type(exc).__name__, exc))
```

Comment:
@Eric89GXL 
While testing I have found that the parenthesis could be in the wrong position maybe due copy-paste typo.
Should I create a new PR for fixing it (if it is a bug) or  can I merge this within my PR?